frontend/explorer/public/js/hash.js

```diff
@@ -1502,8 +1510,10 @@ function appendNavigationMenuBlockstakeOutput(explorerTransactions, hash) {
 function appendNavigationMenuUnlockHash(hash) {
 	var navigation = document.getElementById('nav-links');
 	var unlockSpan = document.createElement('span');
+	var navContainer = document.getElementById('nav-container');
```

Comment:
nav-container does not exist at this point, as you are not creating the block menu here. Why are you not declaring the `nav-container` div in html for all these pages by the way?